Featured & Partner Server Tabs

Starystars67 opened the pull request:
This PR adds the Featured and Partnered servers tabs into the multiplayer menu.

The locales are also included but were generated using Google translate so may be contextually incorrect.

Also removed the second German translation as it appeared to be out of date.

Starystars67 committed:
Enables Featured Servers Tab

Starystars67 committed:
Added translations & removed outdated 2nd de locale

Starystars67 committed:
Added Partner Tab

OfficialLambdax commented on ui/modules/multiplayer/multiplayer.js:
```diff
@@ -1020,6 +1110,7 @@ async function getFeatured() {
 	return new Promise(function(resolve, reject) {
 		var feat = []
 		servers.forEach(server => {
+			console.log(server)
```
leftover debug information?

Starystars67 replied:
Haha, yep, good spot thank you.

Starystars67 committed:
Comment out debug prints

Leaving them there though as helpful for debugging with users when something isn't working as expected

OfficialLambdax commented on ui/modules/multiplayer/multiplayer.js:
```diff
@@ -525,6 +526,7 @@ function($scope, $state, $timeout) {
 	$scope.$on('onServerListReceived', async function (event, data) {
 		servers = await receiveServers(data);
 		featured = await getFeatured();
+		console.log(featured)
```
debug info

Starystars67 replied:
Thank you, fixed too :)

OfficialLambdax reviewed (changes requested):
Still need to test it

Starystars67 committed:
Commented out more debug lines

Starystars67 committed:
Added BG color for partner servers

Starystars67 committed:
Merge branch 'development' into feat-server-tabs

Starystars67 commented:
> Still need to test it

Please could you test and confirm functionality for yourself too, Just as a sanity check for me at least.

Starystars67 commented:
> The featured server tab is empty and the partnered server tab is broken. Thats what happens when i click it ![fghjfgghjghjkghj](https://user-images.githubusercontent.com/91626975/281460408-1bb8e039-5dca-458c-ae9b-c4cb671cdbf1.png)

Just to confirm you cleared cache and reloaded the UI?

OfficialLambdax commented:
Yes i did.. no errors, no nothing

Starystars67 commented:
Odd, okay thank you.

Will load it up again my side and see if I can replicate.

Starystars67 committed:
Fixed partner page not loading.

OfficialLambdax committed:
Merge branch 'development' into feat-server-tabs

OfficialLambdax reviewed (approved):
Appears to be working as intended
